template/java8/model/src/main/java/com/openfaas/model/Request.java

```diff
@@ -52,6 +57,39 @@ public String getQueryRaw() {
 	public Map<String, String> getQuery() {
     	return queryParameters;
     }
+
+    @Override
+    public String getPathRaw() {
+        return pathRaw;
+    }
+
+    @Override
+    public Map<String, String> getPath() {
+        return path;
+    }
+
+    private Map<String, String> parsePathParameters() {
+        Map<String, String> res = new HashMap<String, String>();
```

Comment:
I'll merge, but please see if there is something in stdlib or similar that can do this instead of being so manual. @richierich1610 any ideas on this?